packages/leveldb/src/leveldb/internal.ts

```diff
@@ -36,20 +40,63 @@ export class LevelDBInternal<RxDocType> implements LevelDBStorageInternals<RxDoc
         if (LevelDBInternal.isLevelDBConstructor(this._options)) {
             this.db = this._options.level
         } else {
-            this.db = new Level(this._options.dbName, { valueEncoding: 'json' })
+            this.db = Level(this._options.dbPath, { valueEncoding: 'json' })
```

Comment:
Found 1 bug - level expects a path here

I've done my best to change the types so that things line up, but not sure it's right?